features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx

```diff
@@ -651,126 +646,99 @@ class InfoBar extends React.Component {
-                                                                            defaultMessage: 'Copied',
-                                                                            id: 'Apis.Details.InfoBar.copied',
-                                                                        })
-                                                                        : intl.formatMessage({
-                                                                            defaultMessage: 'Copy to clipboard',
-                                                                            id:
-                                                                                  'Apis.Details.InfoBar.copy.to.clipboard',
-                                                                        })
-                                                                }
-                                                                placement='right'
-                                                            >
-                                                                <CopyToClipboard
-                                                                    text={epUrl}
-                                                                    onCopy={this.onCopy('sandboxUrlCopied')}
+                                                                <Tooltip
+                                                                    title={prodUrlCopied ? 'Copied' : 'Copy to clipboard'}
+                                                                    placement='right'
                                                                 >
-                                                                    <FileCopy color='secondary' />
-                                                                </CopyToClipboard>
-                                                            </Tooltip>
-                                                            <Button
-                                                                variant='contained'
-                                                                size='small'
-                                                                color='primary'
-                                                                className={classes.margin}
-                                                            >
-                                                                <FormattedMessage
-                                                                    id='Apis.Details.InfoBar.test.endpoint'
-                                                                    defaultMessage='Test Endpoint'
+                                                                    <CopyToClipboard
+                                                                        text={epUrl}
+                                                                        onCopy={this.onCopy('prodUrlCopied')}
```

Comment:
This method will automatically call in each render. 
use 
onCopy={() => this.onCopy('prodUrlCopied')} instead.